lib/editor/tinymce/plugins/kalturamedia/tinymce/editor_plugin.js

```diff
@@ -35,46 +35,28 @@
 
         init : function(ed, url) {
             lang = tinyMCE.activeEditor.getParam('language');
+            var ltiUrl = ed.getParam("moodle_plugin_base")+'kalturamedia/tinymce/ltibrowse.php?lang=' + lang;
 
             // Register the command to open a popup window
             ed.addCommand('mcekalturamedia', function() {
 
-                var height = 580;
-                var width = 1100;
-                var panelHeight = height + 90;
-                var panelWidth = width + 23 + 'px';
-                width += 'px';
+                var w = 1200;
+                var h = 700;
+                var dualScreenLeft = window.screenLeft != undefined ? window.screenLeft : screen.left;
+                var dualScreenTop = window.screenTop != undefined ? window.screenTop : screen.top;
 
-                if (Y.UA.ipod !== 0 || Y.UA.ipad !== 0 || Y.UA.iphone !== 0 || Y.UA.android !== 0 || Y.UA.mobile !== null) {
-                    panelWidth = '90%';
-                    width = '100%';
-                }
+                var width = window.innerWidth ? window.innerWidth : document.documentElement.clientWidth ? document.documentElement.clientWidth : screen.width;
+                var height = window.innerHeight ? window.innerHeight : document.documentElement.clientHeight ? document.documentElement.clientHeight : screen.height;
 
-                var iframe = "<iframe id='panelcontentframe' height='" + height + "px' width='" + width + "' src='"+ ed.getParam("moodle_plugin_base")+'kalturamedia/tinymce/ltibrowse.php?lang=' + lang +"'></iframe>";
-                var panelbodycontent = iframe;
-                
-                if (Y.UA.ipod !== 0 || Y.UA.ipad !== 0 || Y.UA.iphone !== 0) {
-                    // This outer div will constrain the iframe from overlapping over its content region on iOS devices.
-                    panelbodycontent = "<div id='panelcontentframecontainer'>" + iframe + "</div>";
-                }    
+                var left = ((width / 2) - (w / 2)) + dualScreenLeft;
+                var top = ((height / 2) - (h / 2)) + dualScreenTop;
+                var newWindow = window.open(ltiUrl, "Browse and Embed", 'scrollbars=no, width=' + w + ', height=' + h + ', top=' + top + ', left=' + left);
```

Comment:
@muli general styling note, row length should be shorted in my opinion (Eclipse rows format)